BGPViewerCore/Service/BGPHeService.cs

```diff
@@ -219,7 +219,17 @@ public IEnumerable<IxModel> GetAsnIxs(int asNumber)
 
         public AsnPrefixesModel GetAsnPrefixes(int asNumber)
         {
-            throw new NotImplementedException();
+            var allHtmlTables = GetDriverWithValidatedResponseFrom($"https://bgp.he.net/AS{asNumber}").FindElements(By.TagName("table"));
```

Comment:
Getting all tables can cause performance loss. Take care with it.

Reply:
It was corrected at [perform some refactorings](https://github.com/wallacemariadeandrade/BGPViewerTool/pull/6/commits/17e9199795a2c24a417ac87bc0bb2542382d4ca1) commit.